src/python/evaluation/common/xlsx_util.py

```diff
@@ -0,0 +1,41 @@
+import logging.config
+from pathlib import Path
+from typing import NoReturn, Union
+
+import pandas as pd
+from openpyxl import load_workbook, Workbook
+
+logger = logging.getLogger(__name__)
+
+
+def remove_sheet(workbook_path: Union[str, Path], sheet_name: str, to_raise_error=False) -> NoReturn:
+    try:
+        workbook = load_workbook(workbook_path)
+        workbook.remove(workbook[sheet_name])
+        workbook.save(workbook_path)
+
+    except KeyError as e:
+        message = f'Sheet with specified name: {sheet_name} does not exist.'
+        if to_raise_error:
+            logger.exception(message)
+            raise e
+        else:
+            logger.info(message)
+
+
+def create_and_get_workbook_path(config) -> Path:
+    workbook = Workbook()
+    workbook_path = config.get_output_file_path()
+    workbook.save(workbook_path)
+    return workbook_path
+
+
+def write_dataframe_to_xlsx_sheet(xlsx_file_path: Union[str, Path], df: pd.DataFrame,
+                                  sheet_name: str, mode='a', index=False):
+    """
+    mode: str Available values are {'w', 'a'}. File mode to use (write or append).
+    index: bool Write row names.
```

Comment:
Should we rename index to to_write_row_names? I think it more readable. Usually bool arguments names start with to or is